pyscal/pyscallist.py

```diff
@@ -68,6 +69,104 @@ def append(self, pyscal_obj):
             raise ValueError
         self.pyscal_list.append(pyscal_obj)
 
+    def df(self):
+        """Dump dataframes of generated relperm data
+
+        Column names are compatible with ecl2df.satfunc. Always uppercase
+        and capillary pressure is PCOW or PCOG (wateroil vs gasoil)
+
+        If the PyscalList contains SCALrecommendations, the CASE column
+        will contain the strings 'pess', 'base' and 'opt' (independent of
+        any alias name potentially used in an input xlsx/csv)
+
+        Returns:
+            pd.DataFrame
+        """
+        gasoil_col_renamer = {"sg": "SG", "krg": "KRG", "krog": "KROG", "pc": "PCOG"}
+        wateroil_col_renamer = {"sw": "SW", "krw": "KRW", "krow": "KROW", "pc": "PCOW"}
+        df_list = []
+        if self.pyscaltype == WaterOilGas:
+            for (satnum, wateroilgas) in enumerate(self.pyscal_list):
+                gasoil_cols = set(wateroilgas.gasoil.table.columns).intersection(
+                    set(["sg", "krg", "krog", "pc"])
```

Comment:
Would rather make sets directly than converting new lists to sets
```suggestion
                    {"sg", "krg", "krog", "pc"}
```

Reply:
Fixed